api/app/graphql/types/user_type.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Types
+  class UserType < Types::BaseObject
+    field :id, ID, null: false
+    field :email, String, null: false
+    field :full_name, String, null: false
+    field :user_tasks, [Types::TaskType], null: true
```

Comment:
I would probably just call this `tasks`. Because user already has a method that will list all its tasks, you wouldn't need the definition `def user_tasks` below. This is Rails's convention over configuration magic.

Reply:
Makes sense!!

https://media.giphy.com/media/12NUbkX6p4xOO4/giphy.gif